packages/site/src/content/InputFile/InputFileNotes.mdx

```diff
@@ -118,7 +126,194 @@ allowed to be uploaded.
 />
 ```
 
+## Basic usage
+
+```tsx
+<InputFile
+  getUploadParams={file => ({
+    url: "https://upload.example.com",
+    key: file.name,
+  })}
+/>
+```
+
+## Default content structure
+
+When no children are provided, InputFile renders different default content based
+on the `variation` and `size` props:
+
+### Dropzone variation (default)
+
+```tsx
+// size="base" (default)
+<InputFile getUploadParams={getUploadParams}>
+  <InputFile.DropzoneWrapper>
+    <Content spacing="small">
+      <InputFile.Button size="small" fullWidth={false} />
+      <InputFile.HintText />
+      <InputFile.Description />
+    </Content>
+  </InputFile.DropzoneWrapper>
+</InputFile>
+
+// size="small"
+<InputFile size="small" getUploadParams={getUploadParams}>
+  <InputFile.DropzoneWrapper>
+    <Content spacing="small">
+      <InputFile.Button size="small" fullWidth={false} />
+      {/* HintText and Description are omitted in small size */}
+    </Content>
+  </InputFile.DropzoneWrapper>
+</InputFile>
+```
+
+### Button variation
+
+```tsx
+<InputFile variation="button" getUploadParams={getUploadParams}>
+  <InputFile.Button fullWidth={true} />
+</InputFile>
+```
+
+## Customization
+
+### Using props
+
+Basic customization can be achieved through props:
+
+```tsx
+<InputFile
+  getUploadParams={getUploadParams}
+  size="small"
+  buttonLabel="Choose Images"
+  allowMultiple={true}
+  allowedTypes="images"
+  hintText="Drop your images here"
+  description="Supported formats: JPG, PNG"
+/>
+```
+
+### Using children
+
+For complete control over the layout, you can provide custom children:
+
+```tsx
+<InputFile getUploadParams={getUploadParams}>
+  <InputFile.DropzoneWrapper>
+    <h2>Upload Your Files</h2>
```

Comment:
Two things,
1. I think this h2 might make more sense to be a `Heading` level 2 to make it clear that this uses Atlantis instead of using base HTML elements
2. I think you should explicitly show the usage in such a way that the Above component is rendered

I also pasted this example into a storybook story and this is what I saw. I think you should wrap these in a content to show it "matching" what a prop driven usage of InputFile 
<img width="1473" alt="image" src="https://github.com/user-attachments/assets/c728d2dc-c3df-47aa-a227-eb7eb1c71189" />